src/VisualStudio/IntegrationTest/TestUtilities/InProcess/LocalsWindow_InProc.cs

```diff
@@ -9,22 +10,71 @@ internal class LocalsWindow_InProc : InProcComponent
     {
         public static LocalsWindow_InProc Create() => new LocalsWindow_InProc();
 
-        public Common.Expression GetEntry(string entryName)
+        public int GetCount()
         {
             var dte = ((DTE2)GetDTE());
             if (dte.Debugger.CurrentStackFrame != null) // Ensure that debugger is running
             {
                 EnvDTE.Expressions locals = dte.Debugger.CurrentStackFrame.Locals;
-                foreach (EnvDTE.Expression local in locals)
+                return locals.Count;
+            }
+
+            return 0;
+        }
+
+        public Common.Expression GetEntry(string entryName)
+        {
+            return new Common.Expression(GetEntryInternal(entryName));
+        }
+
+        public Common.Expression GetEntry(params string[] entryNames)
+        {
+            return new Common.Expression(GetEntryInternal(entryNames));
+        }
+
+        private EnvDTE.Expression GetEntryInternal(params string[] entryNames)
+        {
+            var entry = GetEntryInternal(entryNames[0]);
+            for (int i = 1; i < entryNames.Length; i++)
+            {
+                entry = GetEntryInternal(entryNames[i], entry.DataMembers);
+            }
+
+            return entry;
+        }
+
+        private EnvDTE.Expression GetEntryInternal(string entryName, EnvDTE.Expressions expressions)
+        {
+            // Need to collect nested expressions separately because Expressions cannot be converted to IEnumerable<Expresssion>.
```

Comment:
Thank you! Fixed

---
In reply to: [116634198](https://github.com/dotnet/roslyn/pull/19529#discussion_r116634198) [](ancestors = 116634198)